pkg/apiserver/authentication/internaloauth/interfaces.go

```diff
@@ -0,0 +1,41 @@
+package internaloauth
```

Comment:
Same.

Reply:
> Same.

No.  The previous was a set of combined interfaces and git chose the other half of the interfaces as the move.